spring-boot/admin/src/main/resources/templates/admin/fragments/addOrderingToList.html

```diff
@@ -0,0 +1,66 @@
+<html xmlns:th="http://www.thymeleaf.org" xmlns:sec="http://www.thymeleaf.org/thymeleaf-extras-springsecurity4"
+      xmlns="http://www.w3.org/1999/xhtml">
+<body>
+
+<div th:fragment="addOrderingToPublicServicesList">
+    <script>
+        $(document).ready(function () {
+            $('table[class*="default-data-table"]').DataTable({});
+        });
+    </script>
+</div>
+
+<div th:fragment="addOrderingToAdminApiKeyList">
```

Comment:
This fragment seems to be used in exactly one place, in that case a fragment seems to lead to an unnecessarily complex structure. 

However, maybe the "target columns that should not be orderable" could be passed as parameters to the fragment, to make the fragment reusable for all tables?

Reply:
Done. I extracted one reusable JS function to resolve the code duplication.